src/Calendar.js

```diff
@@ -47,21 +57,15 @@ export default class Calendar extends Component {
       date,
       weekOffset,
       renderDay,
-      onNextMonth,
-      onPrevMonth,
-      onPickDate,
-      onChange
+      renderHeader,
+      onPickDate
     } = this.props;
 
+    const { handlePrevMonth, handleNextMonth } = this;
+
     return (
       <div className="Calendar">
-        <div className="Calendar-header">
-          <button onClick={this.handlePrevMonth}>«</button>
-          <div className="Calendar-header-currentDate">
-            {date.format('MMMM YYYY')}
-          </div>
-          <button onClick={this.handleNextMonth}>»</button>
-        </div>
+        {renderHeader({ date, handlePrevMonth, handleNextMonth })}
```

Comment:
nit: I want the public props to be prefixed with on* not handle*

Reply:
Unsure how you mean that? Should the default callback for renderHeader be using `({ date, onPrevMonth, onNextMonth }) => (` instead?

Reply:
Yes, the props passed to renderHeader should be `{ date, onPrevMonth, onNextMonth }` 😀

Reply:
fixed :)